src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(number);
```

Comment:
```suggestion
      callback.call(this, number);
```
after use this in all operations methods when method will be pass as argument to operate method - it will lose this.
for fix it we can use call method.

Reply:
Oh, sorry.
I never use callback.call()
 I can't understand how it works